### PR TITLE
chore(flag): for rule page

### PR DESCRIPTION
**Pull request by jeffkwoh**
Feature docs: https://www.notion.so/81108939d0f847938385942a16d1305d

### CODE REVIEW

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -791,6 +791,8 @@ def create_partitioned_queues(name):
     "organizations:advanced-search": True,
     # Enable obtaining and using API keys.
     "organizations:api-keys": False,
+    # Enable audit page. ( https://www.notion.so/sentry/Audit-Page-Design-Doc-81108939d0f847938385942a16d1305d)
```
Comment (leedongwei) [inline src/sentry/conf/server.py]:
```suggestion
    # Enable audit page. (https://www.notion.so/sentry/Audit-Page-Design-Doc-81108939d0f847938385942a16d1305d)
```

Comment (leedongwei) [reply src/sentry/conf/server.py]: Do we really need the link to the Notion doc? Feels like `audit page` is a sufficient descriptor.

Comment (leedongwei) [reply src/sentry/conf/server.py]: Don't think we should include it because e298cf1 showed that the link is mutable.

Comment (jeffkwoh) [reply src/sentry/conf/server.py]: We can remove it to keep it consistent with the code base. 

The 'perma' part of the link: `81108939d0f847938385942a16d1305d` doesn't change. The link update was more cosmetic. 
From manual testing: `https://www.notion.so/81108939d0f847938385942a16d1305d` seems to be the perma link for the page. I have not dived into docs to see if this behaviour is documented.

**Comment by jeffkwoh** [comment]
Feature was recently renamed from `audit page` to `rule page` today

**Review by dashed — APPROVED** [review]
Looks good to me 👍

**Review by matejminar — APPROVED** [review]

**Review by billyvg — APPROVED** [review]

**Review by leedongwei — APPROVED** [review]

**Comment by jeffkwoh** [comment]
Thank you all for LGTM!

**Pull request merged**